README.md

```diff
@@ -142,16 +155,13 @@ The following table shows all beans with their respective name (for the `example
 | Bean Name                              | Bean Type                       | Configures by default      |
 |----------------------------------------|---------------------------------|----------------------------|
 | `accessToken` (no client prefix!)      | `AccessTokens`                  | OAuth settings             |
+| `exampleHttpMessageConverters`         | `HttpMessageConverters`         | Text and JSON              |
 | `exampleHttpClient`                    | `HttpClient`                    | Interceptors               |
 | `exampleClientHttpRequestFactory`      | `ClientHttpRequestFactory`      | Timeouts                   |
-| `exampleHttpMessageConverters`         | `HttpMessageConverters`         | Text and JSON              |
-| `exampleRestTemplate`                  | `RestTemplate`                  | Base URL and error handler |
-| `exampleRestTemplate`                  | `StupsOAuth2RestTemplate`       | Base URL and error handler |
-| `exampleRest`                          | `Rest`                          |                            |
+| `exampleRestTemplate`                  | `RestTemplate`                  | Base URL                   |
```

Comment:
If we drop support for it completely we could simplify the whole graph even more, couldn't we?